holoviews/tests/plotting/bokeh/test_server.py

```diff
@@ -163,6 +163,9 @@ def test_server_dynamicmap_with_stream(self):
 
         cds = session.document.roots[0].select_one({'type': ColumnDataSource})
         self.assertEqual(cds.data['y'][2], 2)
+        def loaded():
+            state._schedule_on_load(doc, None)
+        doc.add_next_tick_callback(loaded)
```

Comment:
Should we only add this if we are on Panel 1.4? Will this test now fail if we are using Panel 1.3?

Reply:
Nope, it's safe either way.